extension/content.js

```diff
@@ -170,6 +170,14 @@ function addPatchDiffLinks() {
 	`);
 }
 
+function removeDiffSigns() {
+	$('.blob-code-deletion .blob-code-inner, .blob-code-addition .blob-code-inner').each((index, element) => {
+		const textNode = element.childNodes[0];
+
+		textNode.nodeValue = textNode.nodeValue.replace(/^[+-]/, ' ');
```

Comment:
It should be replaced with an empty string. Otherwise the copied code has an indentation with one space then the actual indentation.

When copying line 314-316 I get the following when pasting in Sublime:

<img width="681" alt="screen shot 2016-07-02 at 22 33 58" src="https://cloud.githubusercontent.com/assets/170270/16542371/1acade32-40a5-11e6-93f7-827c437b91b6.png">